upgrade/upd_2.5.11-to-2.5.12/index.php

```
@@ -168,6 +169,56 @@ public function apply_widenbannerclientpasswd()
+     */
+    public function check_addsessioncookieprefs()
+    {
+        $sql = 'SELECT COUNT(*) FROM `' . $GLOBALS['xoopsDB']->prefix('config')
+            . "` WHERE `conf_name` IN ('session_cookie_samesite', 'session_cookie_secure')";
+        $result = $GLOBALS['xoopsDB']->query($sql);
+        if (!$GLOBALS['xoopsDB']->isResultSet($result) || !($result instanceof \mysqli_result)) {
+            return false;
+        }
+        $row = $GLOBALS['xoopsDB']->fetchRow($result);
+        return $row && (int) $row[0] >= 2;
+    }
```

PHPMD (2.15.0)

[error] check_addsessioncookieprefs accesses the super-global variable $GLOBALS. (undefined)

(Superglobals)